tests/evm/test_error_return_data_out_of_bound.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+
+from zkevm_specs.util import Word
+from zkevm_specs.evm_circuit import (
+    ExecutionState,
+    StepState,
+    verify_steps,
+    Tables,
+    CallContextFieldTag,
+    Block,
+    Bytecode,
+    RWDictionary,
+)
+from zkevm_specs.util.param import MAX_U64
+
+
+TESTING_DATA = (
+    ## is_root is True
+    # is_data_offset_u64_overflow is True
+    (MAX_U64 + 1, 0, True),
+    # is_end_u64_overflow is True
+    (MAX_U64, 1, True),
+    (1, MAX_U64, True),
+    (0, MAX_U64 + 1, True),
+    # is_end_over_return_data_len, return_data_length is set to fixed 320
```

Comment:
Suggestion:
`    # is_end_over_return_data_len is True (as we set return_data_length to 320)
`

Reply:
fixed in [ece414f](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/479/commits/ece414f7f3f3ba7c0b34ab56fffb9cd29be56828)